Resources/Prototypes/Loadouts/loadout_groups.yml

```diff
@@ -50,6 +50,7 @@
   - SilverRing # DeltaV
   - Cane # DeltaV
   - WhiteCane #DeltaV
+  - CDDogtags #CD
```

Comment:
their format is `# _CD`

Reply:
Is that with the underscore? Because when I look at the comments in their loadout_groups.yml it's just `# CD` with a space in between?

Reply:
_CD is for folders not comments afaik